homeassistant/components/camera/push.py

```diff
@@ -39,6 +43,7 @@
     vol.Optional(CONF_TIMEOUT, default=timedelta(seconds=5)): vol.All(
         cv.time_period, cv.positive_timedelta),
     vol.Optional(CONF_IMAGE_FIELD, default='image'): cv.string,
+    vol.Optional(CONF_TOKEN): vol.All(cv.string, vol.Length(min=8)),
```

Comment:
This should be required

Reply:
I made it optional, since the implementation allows the use of the normal authentication.